app/liquid_tags/stackexchange_tag.rb

```diff
@@ -1,5 +1,9 @@
 class StackexchangeTag < LiquidTagBase
   PARTIAL = "liquids/stackexchange".freeze
+  REGISTRY_REGEXP = %r{https://(?<subdomain>\w*)(?:\.)?(?:stackexchange.com|stackoverflow.com)/(?<post_type>q|a|questions)/(?<id>\d{1,20})(?:/)?(?:[\w\-]*)}
+  ID_REGEXP = /\A(?<id>\d{1,20})\Z/
+  SITE_REGEXP = /(?<subdomain>\b[a-zA-Z]+\b)/
+  REGEXP_OPTIONS = [REGISTRY_REGEXP, ID_REGEXP, SITE_REGEXP].freeze
   # update API version here and in stackexchange_tag_spec when a new version is out
   API_URL = "https://api.stackexchange.com/2.2/".freeze
```

Comment:
I checked the current API version; it is 2.3. However, the specs in `spec/liquid_tags/stackexchange_tag_spec.rb` failed because `VCR.use_cassette("stackexchange_tag_stackoverflow")` needs to also be updated to that version.

However, I need help locating where to make that change. 🙏🏾

Reply:
You have several options which get outlined in the error message when trying to run the spec. In this case I think you should follow this approach:

> The current record mode (:once) does not allow new requests to be recorded
           to a previously recorded cassette. You can delete the cassette file and re-run
           your tests to allow the cassette to be recorded with this request